src/customer-stories.html

```diff
@@ -0,0 +1,337 @@
+        <div class="sticky-top stories-toc" tabindex="1">
+          <div class="d-none d-lg-flex flex-column gap-3 mb-md-4 mb-3">
+            <span class="small-caps border-0 bg-transparent fw-bold p-0">Topics</span>
+            <ul class="list-unstyled d-flex flex-column gap-3">
+              <li><a class="link-sidebar fs-7 link-tertiary" href="#stories-GTFS-VCTC">GTFS</a></li>
+              <li>
+                <a class="link-sidebar fs-7 link-tertiary" href="#stories-contactless-payments-GCTD">Contactless Payments</a>
+              </li>
+              <li><a class="link-sidebar fs-7 link-tertiary" href="#stories-benefits-MST">Benefits</a></li>
+            </ul>
+          </div>
+          <div class="d-flex flex-column gap-3 mb-md-4 mb-3">
+            <span class="small-caps border-0 bg-transparent fw-bold p-0">On this page</span>
+            <ul class="list-unstyled d-flex flex-column gap-3">
+              <li>
+                <a class="link-sidebar fs-7 link-tertiary" href="#stories-GTFS-VCTC">Ventura County Transportation Commission</a>
+              </li>
+              <li>
+                <a class="link-sidebar fs-7 link-tertiary" href="#stories-contactless-payments-GCTD">Glenn County</a>
+              </li>
+              <li>
+                <a class="link-sidebar fs-7 link-tertiary" href="#stories-benefits-MST">Monterey-Salinas Transit</a>
+              </li>
+            </ul>
+          </div>
+          <div class="d-flex flex-column gap-3">
+            <span class="small-caps border-0 bg-transparent fw-bold p-0">Contact</span>
+            <p><a class="fs-7 link-tertiary" href="mailto:hello@calitp.org">hello@calitp.org</a></p>
+          </div>
+        </div>
+      </div>
+      <section class="col-11 order-lg-1 offset-lg-2 col-lg-6 pt-3 pt-lg-0">
+        <h2>Selected agency highlights</h2>
+        <div class="ventura">
+          <figure class="image pt-3 mt-3 pb-3">
+            <img
+              alt="Commuter riders board a Ventura County Transportation Commission bus."
+              class="rounded-2 w-100"
+              src="/images/stories-photo-vctc.jpg"
+            />
+            <figcaption class="small text-end text-secondary pt-2">
+              Courtesy of Ventura County Transportation Commission
+            </figcaption>
+          </figure>
+          <figure class="quote border-yellow-4 mt-3">
+            <blockquote>
+              “As a small agency who doesn’t have the resources to be specialists in these areas, we’re able to leverage Cal-ITP’s
+              support to get complicated tech projects done more quickly…or at all.”
+            </blockquote>
+            <figcaption class="ps-3">— Public Transit Director in Ventura County</figcaption>
+          </figure>
+          <h4 class="small-caps border-0 bg-transparent fw-bold text-primary p-0 mb-2" id="stories-GTFS-VCTC">GTFS</h4>
+          <h3>Ventura County Transportation Commission</h3>
```

Comment:
I think semantically, these two headings need to be flipped, e.g. 

* `GTFS` / topic --> `h3`
* `VCTC` / agency --> `h4`

E.g. because VCTC is just one example of GTFS...

OR

Make this `GTFS` heading a `<span>` like the other topic headings. Either way, we should apply the same logic to all 3 sections.

Reply:
So I tried changing the code to a structure where the `TOPIC` is an `h3` and the `Agency Name` is an `h4`. Then I ran the app in the `pa11y` https://pa11y.org/ test locally. And this actually returns an error, because then there's a Heading without any content under it. It's a Heading with another Heading under it. 

```
 • Error: Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2
   ├── #stories-contactless-payments-GCTD
   └── <h3 class="small-caps border-0 bg-transparent fw-bold text-primary p-0 mb-2" id="stories-contactless-payments-GCTD"></h3>
```

So I'm going to go with: `TOPIC` as the `h3` and `Agency Name` as a _span_. And each agency/topic section will be wrapped in its own `<section>`, and the whole thing will be wrapped in `<article>`.

Reply:
Wow, much trickier than I expected!